AUTHORS

```diff
@@ -150,7 +152,6 @@ Kevin Conner <connerke@oregonstate.edu>
 Kevin Lee <kevinlee963@gmail.com>
 Kevin Thomas <kevintab@tutanota.com>
 Kiran Konduru <kiranckonduru@gmail.com>
-Kyriaki Velliniati <kiriaki.vel@gmail.com>
```

Comment:
This is removed since it was present twice - https://github.com/oppia/oppia/pull/10236/files#diff-3d350169560e75d0cf9fc8e3574a3639R160

Reply:
> Kristin Anthony <anthkris09@gmail.com>	

Do you mean `Kristin Anthony` & `Kyriaki Velliniati` are the same person?

Reply:
Sorry, the link was redirecting to a different line!